insert into arrtest_f values(3,'cat1',1.18);
ERROR:  42P01: relation "arrtest_f" does not exist
LINE 1: insert into arrtest_f values(3,'cat1',1.18);
                    ^
